rqes-data-classes/src/commonMain/kotlin/at/asitplus/rqes/collection_entries/QCertCreationAcceptance.kt

```diff
@@ -77,6 +81,12 @@ data class QCertCreationAcceptance(
 
 ) : TransactionData {
 
+    override fun toBase64UrlString(): JsonPrimitive =
```

Comment:
The function is named `to...String()` but produces a `JsonPrimitive`, that's not ideal